getSimulationPA.m

```diff
@@ -6,7 +6,7 @@
     % Set options for sigma point routine
     nderiv = nargout-2;
     op_SP.nderiv = nderiv;
-    op_SP.req = [1,0,0,0,0,1,0]; % [1,1,0,0,0,1,0];
+    op_SP.req = [1,1,0,0,0,1,0]; % [1,1,0,0,0,1,0];
```

Comment:
The second component computes the covariance... I don't really see, where and why we should need it...